src/app/features/shared/components/molecules/UserInfo/UserInfo.tsx

```diff
@@ -10,17 +10,47 @@ type Props = {
   showAsListItem?: boolean
 }
 
-const linkStyle = "{ height: 54px"
+const UserDisplayStyle = styled.div`
+  
+  height: 54px;
+  padding: 12px;
+  flex: 1 0 100%;
+  height: auto;
+  padding: 12px 16px 0;
+
+  >span {
+    vertical-align: text-top;
+  }
+
+  svg {
+    margin-right: 10px;
+    transform: translateY(2px);
+  }
+
+  @media screen and ${ breakpoint("min-width", "laptopM") } {
+    display: inline-block;
+  }
+`
+const linkStyle = `{ 
+  height: 54px; 
+  font-weight: normal; 
+  padding: 12px 16px 9px;
+}`
 
 const UserInfo: React.FC<Props> = ({ showAsListItem = false }) => {
   const { keycloak } = useKeycloak()
   const onClick = () => keycloak.logout()
+  const userDisplay = keycloak.tokenParsed?.name ?? "-"
```

Comment:
We don't want to hide the component all together in this situation?

Reply:
Yes, fixed it